.github/workflows/dependabot-prs.yml

```diff
@@ -17,15 +17,18 @@ jobs:
       # a) it has the `development-dependencies` label, which we add for certain
```

Comment:
This dependabot setup differs from the norm. It is missing the steps:

- Get unique committers
- Disable auto-merge if there are commits from someone other than Dependabot
- Add the Needs QA label to dependabots after any change by someone other than the dependabot bot

I believe I set it up like this a while back by guidance from @JacobEvelyn . IIRC, I think we omitted these because it wasn't needed. Additionally, we don't use buildkite here since the code is small enough to rely on github actions.

Reply:
@cyrorama if we're assigning these out it seems a little strange to me that we wouldn't disable auto-merging if someone had committed on a PR here. I'm not clear on what we gain from not doing those steps really. Any thoughts?